edx_proctoring/models.py

```diff
@@ -196,6 +200,14 @@ def delete(self, *args, **kwargs):  # pylint: disable=arguments-differ
         """
         raise NotImplementedError()
 
+    def __str__(self):
+        """ String representation """
```

Comment:
Was there a warning about these being missing?  Doesn't seem like it should have been needed for Django 2.2 compatibility.

Reply:
```
/py27-django111-drf38/lib/python2.7/site-packages/six.py", line 935, in python_2_unicode_compatible
    klass.__name__)
ValueError: @python_2_unicode_compatible cannot be applied to ProctoredExamReviewPolicyHistory because it doesn't define __str__().
```